core/DataAccess/ArchiveTableDao.php

```diff
@@ -60,7 +60,7 @@ public function getArchiveTableAnalysis($tableDate)
                        SUM(CASE WHEN name NOT LIKE 'done%' THEN 1 ELSE 0 END) AS count_numeric_rows,
                        0 AS count_blob_rows
                   FROM `$numericTable`
-              GROUP BY idsite, date1, date2, period";
+              GROUP BY idsite, date1, date2, period ORDER BY idsite, period, date1";
```

Comment:
I know `date1` and `period` determines `date2`, but I still think there's no harm in also ordering by `date2` since we're grouping by it to prevent any future ordering issues.

```suggestion
              GROUP BY idsite, date1, date2, period ORDER BY idsite, period, date1, date2";
```